server/src/services/dependencyService.ts

```diff
@@ -67,7 +67,7 @@ export class DependencyService {
         module: tsModule
       };
     } else {
-      const workspaceTSPath = path.resolve(workspacePath, 'node_modules/typescript');
+      const workspaceTSPath = path.resolve(workspacePath, tsSDKPath || 'node_modules/typescript');
```

Comment:
Not sure if this kind of fallback actually works. We would have to know what is the default value of the `typescript.tsdk` setting

Reply:
The `typescript.tsdk` setting points directly to the `lib` folder, while we need to point to the parent to do `import(workspaceTSPath)`. This change makes it work.
If the `typescript.tsdk` is unset it defaults to `null`. So this would fallback to the old behaviour and try to load typescript from `node_modules/typescript`.

```suggestion
      const tsPath = tsSDKPath && path.join(tsSDKPath, '..');
      const workspaceTSPath = path.resolve(workspacePath, tsPath || 'node_modules/typescript');
```